core/map.go

```diff
@@ -338,26 +338,30 @@ type Map interface {
 	PutTransient(key interface{}, value interface{}, ttl time.Duration) (err error)
 
 	// AddEntryListener adds a continuous entry listener for this map.
-	// To receive an event, you should implement a corresponding interface for that event such as
-	// EntryAddedListener, EntryRemovedListener, EntryUpdatedListener.
+	// To receive an event, listener should implement a corresponding interface for that event such as
+	// EntryAddedListener, EntryRemovedListener, EntryUpdatedListener,EntryEvictedListener,
+	// EntryMergedListener, MapEvictedListener, MapEvictedListener.
```

Comment:
`MapEvictedListener` is written twice in this one and the following ones. `MapCleared` is missing.